release-process-documentation/documentation-guides/update-release-docs-new.md

```diff
@@ -1,88 +1,180 @@
 # Kubernetes.io Docs Release Playbook
 
-author(s): [chenopis@](mailto:chenopis@google.com), stevepe@
+author(s): @chenopis, @steveperry-53
+revisions: @bradamant3, @zacharysarah
 
+This doc shows how to release documentation during a Kubernetes minor version (for example, v1.10) release.
 
-## TL;DR
+## Introduction
 
-This doc contains an outline of tasks to be completed during a Kubernetes.io docs release cycle for a Kubernetes minor release, e.g. v1.9.
+Kubernetes releases minor versions on a quarterly cadence. For example, in 2017, the releases were: v1.6 (Q1/Jan-Mar), v1.7 (Q2/Apr-Jun), v1.8 (Q3/Jul-Sep), and v1.9 (Q4/Oct-Dec).
 
-## Quarterly release cycle
+SIG Docs is part of the Kubernetes release process: its [maintainers](https://github.com/orgs/kubernetes/teams/sig-docs-maintainers) support documentation created/updated for releases and published on the [kubernetes website](https://kubernetes.io/).
```

Comment:
I've only ever seen it called sig-docs, not SIG Docs.

Why not "created or updated"?

Reply:
Maybe it should be called sig-docs.